apps/etterlatte-saksbehandling-ui/client/src/components/oppgavebenk/components/HandlingerForOppgave.tsx

```diff
@@ -107,7 +113,11 @@ export const HandlingerForOppgave = ({ oppgave }: { oppgave: OppgaveDTO }) => {
         )
       )
     case Oppgavetype.AKTIVITETSPLIKT:
-      return erInnloggetSaksbehandlerOppgave && <AktivitetspliktInfoModal oppgave={oppgave} />
+      return (
+        erInnloggetSaksbehandlerOppgave && (
+          <AktivitetspliktInfoModal oppgave={oppgave} oppdaterStatus={oppdaterStatus} />
+        )
```

Comment:
Er dette egentlig ideelt å ha som en `Modal`? 
Tittet kjapt på det i dev og for meg fremstår det som noe som burde vært en egen side med mer informasjon. 

"Har bruker aktivitetsplikt" ... " født i 1963 eller tidligere". 
Allerede på dette punktet må SB enten ha nettopp vært i behandlingen og ha dette ferskt i minnet, eller lukke modalen, gå inn på personen, åpne modalen _igjen_ og så svare riktig.

Reply:
Grunnen til at valget falt for en modal var at det var raskeste løsning på det tidspunktet vi trengte den dataen om bruker.
Dette er info som nå ligger i førstegangsbehandling og revurdering, men siden det ble lagt til i starten av juni, så har vi ikke data for de 5-6 første månedene med saksbehandling av oms. Denne vil da kunne fjernes så fort vi har data om aktivitetsplikt for alle førstegangsbehandlinger.